raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -481,13 +481,13 @@ private void becomeFollower(OptionalInt leaderId, int epoch) throws IOException
         }
     }
 
-    private VoteResponseData buildVoteResponse(Errors topLevelError, boolean voteGranted) {
-        return buildVoteResponse(topLevelError, Errors.NONE, voteGranted);
+    private VoteResponseData buildVoteResponse(boolean voteGranted) {
```

Comment:
nit: not sure we need this. Maybe it is ok for the caller to be explicit when the error code should be `NONE`?